db/migrate/20170504213048_remove_postal_code_from_targets.rb

```diff
@@ -0,0 +1,23 @@
+class RemovePostalCodeFromTargets < ActiveRecord::Migration
+  class Call < ActiveRecord::Base; end
+  class Plugins::CallTool < ActiveRecord::Base; end
+
+  def up
+    batch_size = 300
+    Call.find_in_batches(batch_size: batch_size) do |group|
+      group.each do |call|
+        target = call.read_attribute(:target)
+        target.delete('postal_code')
+        call.update!(target: target)
+      end
+    end
+
+    Plugins::CallTool.find_in_batches(batch_size: batch_size) do |group|
+      group.each do |call_tool|
+        targets = call_tool.read_attribute(:targets)
+        targets.each { |t| t.delete('postal_code') }
+        call_tool.update!(targets: targets)
+      end
+    end
+  end
+end
```

Comment:
Yes, please do move it into a separate script. I'd also feel much happier with your data changes happening outside of the deployment process. Thanks!